aspnetcore-webapi/APIAnalyzers/WebAPIAnalyzers/FakeRepository.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace WebAPIAnalyzers
+{
+    public class FakeRepository : IFakeRepository
+    {
+        public bool TryGetEmployee(int id, out Employee? employee)
+        {
+            employee = GetEmployees().FirstOrDefault(e => e.Id == id);
+            return employee != null;
+        }
+
+        public List<Employee>? Employees { get; set; }
+
+        public IEnumerable<Employee> GetEmployees()
+        {
+            Employees = new()
+            {
+                new Employee()
+                {
+                    Id = 1,
+                    Name = "John Doe"                },
```

Comment:
Just a small note, can you please fix the indentation/formatting of this bracket?

Reply:
corrected :)